packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -384,6 +384,7 @@ async function startCompass(opts: StartCompassOptions = {}): Promise<Compass> {
   process.env.DEBUG = `${process.env.DEBUG ?? ''},mongodb-compass:main:logging`;
   process.env.MONGODB_COMPASS_TEST_LOG_DIR = path.join(LOG_PATH, 'app');
   process.env.CHROME_LOG_FILE = chromedriverLogPath;
+  process.env.COMPASS_SHOW_NEW_SIDEBAR = 'true';
```

Comment:
Temporary until we flip the switch. Or should I flip the switch in this same PR?

Reply:
(Downside in the interim is that we lose tests for the old sidebar while that's still the one users see)

Reply:
I think flipping the switch in the same pr makes sense to me, that way we know what's shown on main is what's tested.